packages/main/src/CalendarHeader.hbs

```diff
@@ -13,32 +13,31 @@
 		<div
 			data-ui5-cal-header-btn-month
 			class="ui5-calheader-arrowbtn ui5-calheader-middlebtn"
-			?hidden="{{isMonthButtonHidden}}"
+			?hidden="{{_isHeaderMonthButtonHidden}}"
 			tabindex="0"
 			role="button"
 			aria-label="{{accInfo.ariaLabelMonthButton}}"
```

Comment:
Seems to me that this `aria-label` attribute is redundant. For comparison the year button doesn't have such label.
The native behavior of the screen readers would be to announce the texts from the inner `span` elements. That should work as there is no `aria-labelledby` attribute to override this native behavior.

Reply:
As we discussed, we can create another PR with acc improvement around the header.